### DISCUSSION TITLE
Welcome to trifo-robotics-rev-eng Discussions!

### DISCUSSION THREAD

**Discussion by VictorDrijkoningen** [VictorDrijkoningen/trifo-robotics-rev-eng#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VasanK16** [comment]
Hi Victor!
I have been using Trifo Ironpie m6 from June 2020 and it works great but suddenly from Jan 2025 I lost the access to the app where I lost access to Camera, navigation, scheduling the cleaning time, controlling the suction capacity and many other features. 
I will be happy in helping with this project
